dotnet/example_code_legacy/README.md

```diff
@@ -0,0 +1,42 @@
+# README for AWS SDK for .NET Documentation Examples - Legacy code examples
+
+This is a collection of legacy code examples for the AWS SDK for .NET public documentation.  Legacy code examples may use older versions of the .NET framework, and they may not contain unit tests.  
+
+You can find new and updated code examples in the folder example_code.  Legacy code examples may be updated and moved from the example_code_legacy to the example_code folder. 
```

Comment:
...in the **example_code folder**. ... **might** be updated...

Reply:
Meh